qiskit_experiments/database_service/utils.py

```diff
@@ -23,28 +24,21 @@
 import json
 
 import dateutil.parser
-import pkg_resources
 from dateutil import tz
 
-from qiskit.version import __version__ as terra_version
-
 from qiskit_ibm_experiment import (
     IBMExperimentEntryExists,
     IBMExperimentEntryNotFound,
 )
 
 from .exceptions import ExperimentEntryNotFound, ExperimentEntryExists, ExperimentDataError
-from ..version import __version__ as experiments_version
 
 LOG = logging.getLogger(__name__)
 
 
 def qiskit_version():
     """Return the Qiskit version."""
-    try:
-        return pkg_resources.get_distribution("qiskit").version
-    except Exception:  # pylint: disable=broad-except
-        return {"qiskit-terra": terra_version, "qiskit-experiments": experiments_version}
+    return importlib.metadata.distribution("qiskit").version
```

Comment:
Should this not also return the Qiskit Experiments version like the previous code did in the `except` clause? I'm not quite sure what this function is meant for.